packages/editor/src/state/atoms.ts

```diff
@@ -20,8 +20,8 @@ import {
 } from "recoil";
 import { v4 as uuid } from "uuid";
 import { layoutModel } from "../App.js";
-import { RenderState } from "../worker/message.js";
-import OptimizerWorker from "../worker/OptimizerWorker.js";
+import { RenderState } from "../worker/common";
+import OptimizerWorker from "../worker/OptimizerWorker";
```

Comment:
Looks like the `.js` suffix got lost here; @wodeni I forget, do we still want to use this? I feel like I remember having some way to automatically lint this in the past, but we got rid of it for some reason.